CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 0.7.0
+
```

Comment:
I would mention here there all the changes are documented on our website and ideally provide a link here.

Reply:
So, I don't think we document 'the changes'. What about something like: "see our website for up-to-date documentation"?

Reply:
I don't think this has been resolved yet, has it?